build/yamls/antrea-node-init.yml

```diff
@@ -0,0 +1,42 @@
+kind: DaemonSet
+apiVersion: extensions/v1beta1
+metadata:
+  name: antrea-node-init
+  namespace: kube-system
+  labels:
+    app: antrea-node-init
+spec:
+  template:
+    metadata:
+      labels:
+        app: antrea-node-init
+    spec:
+      hostPID: true
+      hostNetwork: true
+      containers:
+        - name: node-init
+          image: gcr.io/google-containers/startup-script:v1
+          imagePullPolicy: IfNotPresent
+          securityContext:
+            privileged: true
+          env:
+          - name: STARTUP_SCRIPT
+            value: |
+              #! /bin/bash
+
+              set -o errexit
+              set -o pipefail
+              set -o nounset
+
+              echo "Changing kubelet configuration to --network-plugin=cni --cni-bin-dir=/home/kubernetes/bin"
```

Comment:
Should it check the configuration has been updated to the desired one and skip restarting kubelet? Otherwise kubelet will be restarted every time this pod is restarted?

Reply:
Done